src/Core/View.js

```diff
@@ -512,4 +513,82 @@ View.prototype.normalizedToViewCoords = function normalizedToViewCoords(ndcCoord
     return _eventCoords;
 };
 
+/**
+ * Return objects from some layers/objects3d under the mouse in this view.
+ *
+ * @param {Object} mouse - mouse position in window coordinates (0, 0 = top-left)
+ * @param {...*} where - where to look for objects. Can be either: empty (= look
+ * in all layers with type == 'geometry'), layer ids or layers or a mix of all
+ * the above.
+ * @return {Array} - an array of objects. Each element contains at least an object
+ * property which is the Object3D under the cursor. Then depending on the queried
+ * layer/source, there may be additionnal properties (coming from THREE.Raycaster
+ * for instance).
+ *
+ * @example
+ * view.pickObjectsAt({ x, y })
+ * view.pickObjectsAt({ x, y }, 'wfsBuilding')
+ * view.pickObjectsAt({ x, y }, 'wfsBuilding', myLayer)
+ */
+View.prototype.pickObjectsAt = function pickObjectsAt(mouse, ...where) {
+    const results = [];
+    const sources = where.length == 0 ?
+        this.getLayers(l => l.type == 'geometry') :
+        [...where];
+
+    for (let i = 0; i < sources.length; i++) {
+        if (typeof (sources[i]) === 'string') {
+            const lookup = this.getLayers(l => l.id == sources[i]);
+            if (!lookup.length) {
+                throw new Error(`Invalid layer id used as where argument (value = ${where})`);
+            }
+            sources[i] = lookup[0];
+        }
+    }
+
+    for (const source of sources) {
+        if (source.type === 'geometry') {
+            // does this layer have a custom picking function?
+            if (source.pickObjectsAt) {
+                results.splice(
+                    results.length, 0,
+                    ...source.pickObjectsAt(this, mouse));
+            } else {
+                //   - it hasn't: this layer is attached to another one
+                let parentLayer;
+                this.getLayers((l, p) => {
```

Comment:
if a geometric layer is attached to another geometric layer.
We could affect it the reference of `parentLayer.object3d` in the `GeometryLayer.prototype.attach` function
`layer.object3d = this.object3d`

Who would avoid the party with `it hasn't: this layer is attached to another one`

WDYT?

Reply:
In other places of itowns, layer.object3d is the object used to add layer's mesh to the scene.

So I don't think it make sense for now to store parent layer's object3D here.